src/bikeracks/templates/base.html

```diff
@@ -118,13 +118,61 @@
                       </div>
                       
                   </li>
+                  <li class="nav-item">
+                    <button class="btn btn-secondary" data-toggle="modal" data-target="#guideModal">Guide</button>
```

Comment:
Maybe `Help` or just `?` ?

Or leave a cookie and if they don't have it, just pop up the guide automatically.